packages/react-core/src/components/Form/examples/FormFieldGroups.tsx

```diff
@@ -0,0 +1,324 @@
+import React from 'react';
+import {
+  Form,
+  FormGroup,
+  FormFieldGroup,
+  FormFieldGroupExpandable,
+  FormFieldGroupHeader,
+  TextInput,
+  Button
+} from '@patternfly/react-core';
+import TrashIcon from '@patternfly/react-icons/dist/esm/icons/trash-icon';
+
+export const FieldGroups: React.FunctionComponent = () => {
+  const initialValues = {
+    '1-label1': '',
+    '1-label2': '',
+    'group1-label1': '',
+    'group1-label2': '',
+    'group2-label1': '',
+    'group2-label2': '',
+    'group3-label1': '',
+    'group3-label2': '',
+    'exapnded-group1-label1': '',
+    'exapnded-group1-label2': '',
```

Comment:
Looks like we just need to make this is fixed throughout `FormFieldGroups.tsx`! I'm seeing this in a few places

```suggestion
    'expanded-group1-label2': '',
```